.github/ISSUE_TEMPLATE/community-specialist.yaml

```
@@ -0,0 +1,62 @@
+name: Community Specialist Nomination Template
+description: Nominate yourself or someone else to become a community specialist.
+title: 'Community Reviewer Nomination: <username>'
+labels: nomination/community-specialist
+body:
+  - type: markdown
+    attributes:
+      value: >
+        The Documentation and Community SIG maintains several SIG roles requiring  approval - this nomination form is for serving as a _community specialist_,
+        responsible for reviewing contributions to `o3de.org/community`, `o3de.org/blogs`, `o3de.org/docs/contributing`, and the `o3de/community` repo.
+        Community reviewers are expected to:
+
+        * Assist in the review of blogs and other community-related content.
+        * Help surface and guide discussion for `o3de/community` issues.
+        * Provide feedback on RFCs which affect the O3DE community.
+        * Contribute to and participate in discussion channels like Discord and GitHub forums. Serving as a moderator is recommended but not required.
+        * Serving as a dedicated representative of the broader O3DE community
+        * Assist in enforcing the [code of conduct](LINK).
```

I assume the link is to the LF project's code of conduct, the same one on o3de.org's home page?
```suggestion
        * Assist in enforcing the [code of conduct](https://lfprojects.org/policies/code-of-conduct/).
```